tests/backend/invite/test_base.py

```diff
@@ -337,6 +338,25 @@ async def test_delete(
             pass
 
 
+@pytest.mark.trio
+@pytest.mark.parametrize("is_revoked", [True, False])
+async def test_user_invitation_already_member(alice, bob, backend, alice_backend_sock, is_revoked):
+    with patch.object(User, "is_revoked") as is_revoked_mock:
```

Comment:
I don't understand why the mocks are needed (and mocking the postgresql implementation seems scary given we remove a whole query...)

I think it should be enough to just use the bob fixture (so we have an existing user) then revoke (if the parametrize says so) at the beginning of the test.